modules/images-other-jenkins-config-kubernetes.adoc

```diff
@@ -66,6 +70,7 @@ data:
         <org.csanchez.jenkins.plugins.kubernetes.ContainerTemplate>
           <name>jnlp</name>
           <image>openshift/jenkins-agent-maven-35-centos7:v3.10</image>
+          // REMOVE OR REPLACE jenkins-agent-maven?
```

Comment:
the preceding paragraphs explain that this is still here for legacy explanation reasons.
We can keep it for 4.11.